samples/HTTPParallelRequestDemo/Main.spl

```diff
@@ -0,0 +1,109 @@
+use com.ibm.streamsx.inet.http::*;
+
+/**
+ * This main composite invokes the HTTPRequestAsyncComposite cyclcal
```

Comment:
Seems like an unfinished comment.  Can a full explanation of the sample be added?